tests/testthat/_snaps/build-reference-index.md

```diff
@@ -51,14 +51,13 @@
       data_reference_index(pkg)
     Condition
       Error:
-      ! All topics must be included in reference index
-      x Missing topics: c, e, and ?
-      i Either add to _pkgdown.yml or use @keywords internal
+      ! 3 topics missing from index: "c", "e", and "?"
+      i Edit _pkgdown.yml to fix the problem.
```

Comment:
the hint to use `@keywords internal` was very helpful here

Reply:
The word is a bit more awkward now, but I think that's ok in return for greater consistency.